mixed-content/generic/mixed-content-test-case.js

```diff
@@ -23,6 +23,7 @@ function MixedContentTestCase(scenario, description, sanityChecker) {
 
   var sameOriginHost = location.hostname;
   var crossOriginHost = "{{domains[www1]}}";
+  var localhostIPv4Host = "127.0.0.1";
```

Comment:
I don't think this is going to work in general. @jgraham will have thoughts, I'm sure, but I don't think we can make the assumption that 127.0.0.1 is actually running a server. It won't be running a server on `w3c-test.org`, for instance.